### DISCUSSION TITLE
No iptv on KPN

### DISCUSSION THREAD

**Discussion by Patrickvts** [fabianishere/udm-iptv#113] [Q&A]
Hi,

Yesterday I upgraded my KPN v10 to a UDM Pro, internet works great but IPTV isn't working very well(video starts for 3 sec then stops and giving me an error), since kpn iptv should have worked out of the box, i dont know what is going on.

The decoders are on their own tv vlan (40) @ 10.140.40.0/24 and iptv-class IPTV_RG

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
45: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.206.98.89/20 brd 10.206.111.255 scope global iptv
       valid_lft forever preferred_lft forever
10.206.96.0/20 proto kernel scope link src 10.206.98.89 
213.75.112.0/21 via 10.206.96.1 metric 245 
=== Service Logs ===
Apr 20 10:53:35 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machine.
Apr 20 10:53:35 ubnt udm-iptv[348]: Device iptv already exists.. Deleting device
Apr 20 10:53:35 ubnt udm-iptv[348]: Obtaining IP address for VLAN interface
Apr 20 10:53:35 ubnt udm-iptv[348]: udhcpc (v1.22.1) started
Apr 20 10:53:35 ubnt udm-iptv[348]: Sending discover...
Apr 20 10:53:35 ubnt udm-iptv[348]: Sending select for 10.206.98.89...
Apr 20 10:53:36 ubnt udm-iptv[348]: Lease of 10.206.98.89 obtained, lease time 3600
Apr 20 10:53:36 ubnt udm-iptv[348]: NATing IPTV network ranges (if necessary)
Apr 20 10:53:36 ubnt udm-iptv[348]: Setting up igmpproxy
Apr 20 10:53:36 ubnt udm-iptv[348]: Starting igmpproxy

I hope you can help me,

Patrick

**Comment by fabianishere** [comment]
Have you tried selecting `br40` as LAN interface in the installer?

**Reply by Patrickvts** [reply]
Yea, i did select `br40` in the installer, apparently it didnt save the configuration, I will reconfigure it tonight, hope that fixes it.
